docs/user_guide/write_api/online_writer.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: default
+title: Online Writer (Experimental)
+parent: Write APIs
+grand_parent: User Guides
+permalink: /docs/user_guide/write_api/online_writer
+---
+# [Experimental] Online Writer
```

Comment:
Sorry, one more comment... should we use "Online Producer" consistently? I see the titles say Writer while the body of the text say Producer.

Reply:
Yeah, I've been wondering b/w the two. Will change it to go with "Producer" to keep it consistent with the names in code too.